nidm/nidm-experiment/terms/nidm-experiment.owl

```diff
@@ -10,7 +10,7 @@
 @prefix xsp: <http://www.owl-ontologies.com/2005/08/07/xsp.owl#> .
 @prefix daml: <http://www.daml.org/2001/03/daml+oil#> .
 @prefix meta: <http://purl.obolibrary.org/obo/iao/dev/ontology-metadata.owl#> .
-@prefix nidme: <http://purl.org/nidash/nidm/experiment#> .
+@prefix nidm: <http://purl.org/nidash/nidm/experiment#> .
```

Comment:
i think we are moving towards a single namespace for all nidm terms with a trailing '/':

`@prefix nidm: <http://purl.org/nidash/nidm/> .`